### PR TITLE
Documentation admin panel

### PR DESCRIPTION
**Pull request by ketankartoza**
This PR is for Issue #1461

### CODE REVIEW

**Comment by ketankartoza** [comment]
@dimasciput The reason for the size 8 is changing the structure, updating the older doc, adding new files, discussion, and all.

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 2 min 7 sec

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 1 min 56 sec

**Comment by dimasciput** [comment]
@Jeremy-Prior please review

File: docs/src/administrator/manual/django-table.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+![Django Admin Table](./img/django-table-1.png)
+
+1. **Add button**: Clicking on this button will allow user to add a new record in the table. Clicking on [add](django-add-data.md) will navigate you to add data record documentation.
+
+2. **Search**: Search feature to search specific record of the table.
+
+3. **Toggle select records**: Checkbox to select/deselect records.
+
+4. **Action**: Action dropdown user can choose the action from this dropdown and perform on selected object.
```
Comment (Jeremy-Prior) [inline docs/src/administrator/manual/django-table.md:11]:
```suggestion
4. **Action**: Users can choose the action from this dropdown and perform it on a selected object.
```

File: docs/src/administrator/manual/django-table.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+5. **Go**: Clicking on this button will allow user to perform selected option on selected object.
```
Comment (Jeremy-Prior) [inline docs/src/administrator/manual/django-table.md:13]:
```suggestion
5. **Go**: Clicking on this button will allow users to perform the selected option on the selected object.
```

File: docs/src/administrator/manual/django-table.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+6. **Objects**: Available objects containing records. User can see the details of the object by clicking on the link.
```
Comment (Jeremy-Prior) [inline docs/src/administrator/manual/django-table.md:15]:
```suggestion
6. **Objects**: Available objects containing records. Users can see the details of the object by clicking on the link.
```

File: docs/src/administrator/manual/django-table.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+## Group Table
+
+![Group Table](./img/django-table-2.png)
+
+This table is used to store existing user groups, that will affect available options in the [Report](../../user/manual/explore/reports.md).
+You can assign a user to groups in the User Table.
```
Comment (Jeremy-Prior) [inline docs/src/administrator/manual/django-table.md:22]:
```suggestion
Administrators can assign a user to groups in the User Table.
```

File: docs/src/administrator/manual/django-table.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+1. **Toggle sorting**: Clicking on this icon will allow user to toggle data.
```
Comment (Jeremy-Prior) [inline docs/src/administrator/manual/django-table.md:24]:
```suggestion
1. **Toggle sorting**: Clicking on this icon will allow users to toggle data.
```

File: docs/src/administrator/manual/django-table.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+
+2. **Add Group**: Clicking on the `ADD GROUP` button will allow user to add a new group. Click on [add group](django-add-data.md) to see detailed documentation about adding a new group.
```
Comment (Jeremy-Prior) [inline docs/src/administrator/manual/django-table.md:26]:
```suggestion
2. **Add Group**: Clicking on the `ADD GROUP` button will allow users to add a new group. Click on [add group](django-add-data.md) to see detailed documentation about adding a new group.
```

File: docs/src/administrator/manual/django-table.md [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+
+3. **Edit Record**: Clicking on the object will allow user to change/edit the particular record. Click on [change group](django-change-data.md) to see detailed documentation about editing a group.
```
Comment (Jeremy-Prior) [inline docs/src/administrator/manual/django-table.md:28]:
```suggestion
3. **Edit Record**: Clicking on the object will allow users to change/edit the particular record. Click on [change group](django-change-data.md) to see detailed documentation about editing a group.
```

File: docs/src/administrator/manual/django-table.md
Comment (Jeremy-Prior) [inline docs/src/administrator/manual/django-table.md:1]: I've made comments but this is administrator documentation so some of the instances where `user` is applied should probably be changed to `administrator`

**Review by Jeremy-Prior — CHANGES_REQUESTED** [review]

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 2 min 4 sec

File: docs/src/administrator/manual/django-admin.md [hunk 8]
```diff
@@ -12,72 +12,69 @@ _![Django Admin](./img/admin-1.png)_
...
-1. **Application**: Application or module that exist in this site.
-2. **Table**: Table or model, where the data is stored.
-3. **Recent Actions**: List of activities recently happened in admin site.
+In the top right corner of the admin panel, users will find a section that usually displays user's username. In this section, users have the following options:
 
-To data in a table, click on the table name.
+1. **Welcome**: Displays welcome with admin name.
 
-## Django Admin Table
+2. **View Site**: Clicking on this option will take users to the `SAWPS` website.
```
Comment (Jeremy-Prior) [inline docs/src/administrator/manual/django-admin.md:23]:
```suggestion
2. **View Site**: Clicking on this option will take administrators to the `SAWPS` website.
```

File: docs/src/administrator/manual/django-admin.md [hunk 9]
```diff
@@ -12,72 +12,69 @@ _![Django Admin](./img/admin-1.png)_
...
+
+3. **Change Password**: Clicking on this option allows users to change their admin password. Users will be prompted to provide their current password and set a new one.
```
Comment (Jeremy-Prior) [inline docs/src/administrator/manual/django-admin.md:25]:
```suggestion
3. **Change Password**: Clicking on this option allows administrators to change their administration password. Administrators will be prompted to provide their current password and then set a new one.
```

File: docs/src/administrator/manual/django-admin.md [hunk 10]
```diff
@@ -12,72 +12,69 @@ _![Django Admin](./img/admin-1.png)_
...
+    **Input Fields**
 
-_![Django Admin Table](./img/admin-3.png)_
+    1. **Old Password**: In this field, users should provide the current password associated with the user account. This is a mandatory field.
 
-1. **Table records**: Show the first page of table records.
-2. **Count of selected records**: Shows the number of selected records.
-3. **Toggle select records**: Checkbox to select/deselect records.
-4. **Add new records**: Button to add a new record.
+    2. **New Password**: In this field, users should enter the new password they want to set for their account. The password should meet the following criteria:
```
Comment (Jeremy-Prior) [inline docs/src/administrator/manual/django-admin.md:33]:
```suggestion
    2. **New Password**: In this field, administrators should enter the new password they want to set for their account. The password should meet the following criteria:
```

File: docs/src/administrator/manual/django-admin.md [hunk 11]
```diff
@@ -12,72 +12,69 @@ _![Django Admin](./img/admin-1.png)_
...
+2. **View Site**: Clicking on this option will take users to the `SAWPS` website.
+
+3. **Change Password**: Clicking on this option allows users to change their admin password. Users will be prompted to provide their current password and set a new one.
+
+    ![Change Password](./img/admin-8.png)
+
+    **Input Fields**
 
-_![Django Admin Table](./img/admin-3.png)_
+    1. **Old Password**: In this field, users should provide the current password associated with the user account. This is a mandatory field.
```
Comment (Jeremy-Prior) [inline docs/src/administrator/manual/django-admin.md:31]:
```suggestion
    1. **Old Password**: In this field, administrators should provide the current password associated with their account. This is a mandatory field.
```

File: docs/src/administrator/manual/django-admin.md [hunk 12]
```diff
@@ -12,72 +12,69 @@ _![Django Admin](./img/admin-1.png)_
...
 
-To edit a record, click on the record.
+    * The password can't be too similar to other personal information.
+    * The password must contain at least 12 characters.
+    * The password can't be a commonly used password.
+    * The password can't be entirely numeric.
+    * The password should include the following:
+    * Numeric character
+    * Uppercase letter
+    * Special character (@#%;)
```
Comment (Jeremy-Prior) [inline docs/src/administrator/manual/django-admin.md:42]:
```suggestion
    * The password should include the following:
        * Numeric character
        * Uppercase letter
        * Special character (@#%;)
```

File: docs/src/administrator/manual/django-admin.md [hunk 13]
```diff
@@ -12,72 +12,69 @@ _![Django Admin](./img/admin-1.png)_
...
-_![Django Admin Form](./img/admin-4.png)_
+    3. **New Password Confirmation**: Re-enter the new password in this field to confirm it. It should match the password entered in the `New Password` field.
 
-1. **Form fields**: Form where we can input values for our record.
-2. **Delete button**: Delete currently opened record. It will take you to confirmation page.
-3. **Save and add another**: Save current record, then redirect to a new page to add new record.
-4. **Save and continue editing**: Save current record while still showing current record.
-5. **Save**: Save current record, then redirect to Django Admin Table/record list.
-6. **History**: Button to see actions applied to current record.
+    4. **Change My Password Button**: Once users have filled in the required information, they can click the `CHANGE MY PASSWORD` button to submit the form. If all the input is valid and the new password meets the criteria, the password for the user account will be updated.
```
Comment (Jeremy-Prior) [inline docs/src/administrator/manual/django-admin.md:48]:
```suggestion
    4. **Change My Password Button**: Once administrators have filled in the required information, they can click the `CHANGE MY PASSWORD` button to submit the form. If all the input is valid and the new password meets the criteria, the password for their account will be updated.
```

File: docs/src/administrator/manual/django-admin.md [hunk 14]
```diff
@@ -12,72 +12,69 @@ _![Django Admin](./img/admin-1.png)_
...
 
-## Activities Table
+After successfully changing the password, the user will receive a confirmation message indicating that the password has been updated.
```
Comment (Jeremy-Prior) [inline docs/src/administrator/manual/django-admin.md:50]:
```suggestion
After successfully changing the password, the administrator will receive a confirmation message indicating that the password has been updated.
```

File: docs/src/administrator/manual/django-admin.md [hunk 15]
```diff
@@ -12,72 +12,69 @@ _![Django Admin](./img/admin-1.png)_
...
 
-_![Activities Table](./img/admin-5.png)_
+4. **Log Out**: Clicking on this option will log the user out of the admin panel and return the user to the login page.
```
Comment (Jeremy-Prior) [inline docs/src/administrator/manual/django-admin.md:52]:
```suggestion
4. **Log Out**: Clicking on this option will log the administrator out of the administration panel and return them to the login page.
```

File: docs/src/administrator/manual/django-admin.md [hunk 16]
```diff
@@ -12,72 +12,69 @@ _![Django Admin](./img/admin-1.png)_
...
-    * founder_population
-    * reintroduction_source
-    * intake_permit
-    * offtake_permit
-    * translocation_destination
+    1. **Log in again**: Clicking this option will open the login page.
 
-This is an example of correct **Export fields** value:
-`["translocation_destination", "founder_population"]`
+5. **Recent Actions**: The `Recent Actions` section is typically located on the right side of the admin panel and provides a list of recent actions that have been performed within the admin interface. These actions may include additions, deletions, or changes to data in your project. Clicking on any of the available links will take a user to the detail of recent action.
```
Comment (Jeremy-Prior) [inline docs/src/administrator/manual/django-admin.md:58]:
```suggestion
5. **Recent Actions**: The `Recent Actions` section is located on the right side of the administration panel and provides a list of recent actions that have been performed within the administration interface. These actions may include additions, deletions, or changes to data in the project. Clicking on any of the available links will take an administrator to the details of the recent action.
```

File: docs/src/administrator/manual/django-admin.md [hunk 17]
```diff
@@ -12,72 +12,69 @@ _![Django Admin](./img/admin-1.png)_
...
 
-Any update on this table will be reflected on the [Activity Report](../../user/manual/explore/reports.md).
+6. **Site Administration**: Site Administration this section provides links to various models and apps registered with the admin panel. It serves as the primary navigation point for managing a project's data. Clicking on any of the available links will take a user to the respective model's management page.
 
-## Group Table
+7. **Add**: This button is used to create a new item of the associated model. Clicking on `Add` link will open a form where users can enter the details of the new record.
+
+8. **Change**: This button is used to edit an existing item from the list. When users click the `Change` button, they will be redirected to a page where they can see all the associate data of that model and can select data to change.
+
+* **Table**: To see data in a table, click on the table name.
```
Comment (Jeremy-Prior) [inline docs/src/administrator/manual/django-admin.md:66]: Is this line still necessary? The same information is in line 72

File: docs/src/administrator/manual/django-change-data.md [hunk 18]
```diff
@@ -0,0 +1,36 @@
...
+
+    ![History](./img/django-change-data-2.png)
+
+## Group
+
+![Confirmation](./img/django-change-data-3.png)
+
+Administrators can update the field of the object by changing the value of the fields. E.g. change the permissions of the group and update the description of the group.
+
+The other options(`History`, `Delete`, `Save and add another`, `Save and continue editing`, `Save`) work same as they are working for the other objects.
```
Comment (Jeremy-Prior) [inline docs/src/administrator/manual/django-change-data.md:36]:
```suggestion
The other options (`History`, `Delete`, `Save and add another`, `Save and continue editing`, `Save`) work same for other objects as they do in this example.
```

File: docs/src/administrator/manual/django-table.md [hunk 19]
```diff
@@ -0,0 +1,28 @@
+## Activities Table
+
+![Django Admin Table](./img/django-table-1.png)
+
+1. **Add button**: Clicking on this button will allow user to add a new record in the table. Clicking on [add](django-add-data.md) will navigate you to add data record documentation.
```
Comment (Jeremy-Prior) [inline docs/src/administrator/manual/django-table.md:5]:
```suggestion
1. **Add button**: Clicking on this button will allow administrators to add a new record in the table. Clicking [here](django-add-data.md) will redirect users to add data record documentation.
```

File: docs/src/administrator/manual/django-table.md [hunk 20]
```diff
@@ -0,0 +1,28 @@
...
+
+2. **Search**: Search feature to search specific record of the table.
```
Comment (Jeremy-Prior) [inline docs/src/administrator/manual/django-table.md:7]:
```suggestion
2. **Search**: Feature to allow for searching of a specific object in the table using keywords.
```

File: docs/src/administrator/manual/django-table.md [hunk 21]
```diff
@@ -0,0 +1,28 @@
...
+
+3. **Toggle select records**: Checkbox to select/deselect records.
+
+4. **Action**: Users can choose the action from this dropdown and perform it on a selected object.
```
Comment (Jeremy-Prior) [inline docs/src/administrator/manual/django-table.md:11]:
```suggestion
4. **Action**: Administrators can choose the action from this dropdown and then perform it on a selected object.
```

File: docs/src/administrator/manual/django-table.md [hunk 22]
```diff
@@ -0,0 +1,28 @@
...
+
+5. **Go**: Clicking on this button will allow users to perform the selected option on the selected object.
```
Comment (Jeremy-Prior) [inline docs/src/administrator/manual/django-table.md:13]:
```suggestion
5. **Go**: Clicking on this button will allow administrators to perform the selected option on the selected object.
```

File: docs/src/administrator/manual/django-table.md [hunk 23]
```diff
@@ -0,0 +1,28 @@
...
+
+6. **Objects**: Available objects containing records. Users can see the details of the object by clicking on the link.
```
Comment (Jeremy-Prior) [inline docs/src/administrator/manual/django-table.md:15]:
```suggestion
6. **Objects**: Available objects containing records. Administrators can see the details of the object by clicking on the link (i.e. the object name).
```

File: docs/src/administrator/manual/django-table.md [hunk 24]
```diff
@@ -0,0 +1,28 @@
...
+
+## Group Table
+
+![Group Table](./img/django-table-2.png)
+
+This table is used to store existing user groups, that will affect available options in the [Report](../../user/manual/explore/reports.md).
+Administrators can assign a user to groups in the User Table.
```
Comment (Jeremy-Prior) [inline docs/src/administrator/manual/django-table.md:22]:
```suggestion
This table is used to store existing user groups, that will affect the available options in the [Report](../../user/manual/explore/reports.md).

Administrators can assign a user to groups in the User Table.
```

File: docs/src/administrator/manual/django-table.md [hunk 25]
```diff
@@ -0,0 +1,28 @@
...
+
+1. **Toggle sorting**: Clicking on this icon will allow users to toggle data.
```
Comment (Jeremy-Prior) [inline docs/src/administrator/manual/django-table.md:24]:
```suggestion
1. **Toggle sorting**: Clicking on this icon will allow administrators to toggle the displayed order of the data.
```

File: docs/src/administrator/manual/django-table.md [hunk 26]
```diff
@@ -0,0 +1,28 @@
...
+
+2. **Add Group**: Clicking on the `ADD GROUP` button will allow users to add a new group. Click on [add group](django-add-data.md) to see detailed documentation about adding a new group.
```
Comment (Jeremy-Prior) [inline docs/src/administrator/manual/django-table.md:26]:
```suggestion
2. **Add Group**: Clicking on the `ADD GROUP` button will allow administrators to add a new group. Click [here](django-add-data.md) to see detailed documentation about adding a new group.
```

File: docs/src/administrator/manual/django-table.md [hunk 27]
```diff
@@ -0,0 +1,28 @@
...
+
+3. **Edit Record**: Clicking on the object will allow users to change/edit the particular record. Click on [change group](django-change-data.md) to see detailed documentation about editing a group.
```
Comment (Jeremy-Prior) [inline docs/src/administrator/manual/django-table.md:28]:
```suggestion
3. **Edit Record**: Clicking on the object will allow administrators to change/edit a particular record. Click [here](django-change-data.md) to see detailed documentation about editing a group.
```

**Review by Jeremy-Prior — CHANGES_REQUESTED** [review]

File: docs/src/administrator/manual/django-admin.md (hunk 17 above)
Comment (ketankartoza) [reply docs/src/administrator/manual/django-admin.md:66]: as we are doing the doc in tutorial style I think that having this line is okay

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 5 min 26 sec

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 2 min 36 sec

File: docs/src/administrator/manual/django-admin.md (hunk 17 above)
Comment (Jeremy-Prior) [reply docs/src/administrator/manual/django-admin.md:66]: This is not a tutorial style document, this is the manual which is a page-by-page explanation. Please remove this line as the information is repeated in line 72 and it is more logical for it to be in line 72.

Comment (ketankartoza) [reply docs/src/administrator/manual/django-admin.md:66]: Ah sorry, yes it is manual documentation. I will remove this line.

File: docs/src/administrator/manual/django-admin.md
[the remark this answers is not in the document]
Comment (ketankartoza) [reply docs/src/administrator/manual/django-admin.md:1]: Okay

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 3 min 46 sec

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 2 min 4 sec

**Comment by dimasciput** [comment]
@Jeremy-Prior please review

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 1 min 41 sec

File: docs/src/administrator/manual/django-change-data.md [hunk 28]
```diff
@@ -0,0 +1,36 @@
...
+
+1. **Form fields**: Form where administrators can input values for their record.
+
+2. **Delete button**: Delete currently opened record. It will take administrators to a confirmation page.
+
+    ![Confirmation](./img/django-change-data-1.png)
+
+    1. **Detail**: Details about the object.
+
+    2. **Yes, I'm sure**: Button for confirm deletion of the object.
```
Comment (Jeremy-Prior) [inline docs/src/administrator/manual/django-change-data.md:16]:
```suggestion
    2. **Yes, I'm sure**: Button for confirming the deletion of the object.
```

File: docs/src/administrator/manual/django-change-data.md [hunk 29]
```diff
@@ -0,0 +1,36 @@
...
+
+    3. **No, take me back**: Button to cancel deletion of the object.
```
Comment (Jeremy-Prior) [inline docs/src/administrator/manual/django-change-data.md:18]:
```suggestion
    3. **No, take me back**: Button to cancel the deletion of the object.
```

File: docs/src/administrator/manual/django-table.md [hunk 30]
```diff
@@ -0,0 +1,31 @@
+# Django Tables
+
+## Activities Table
+
+![Django Admin Table](./img/django-table-1.png)
+
+1. **Add button**: Clicking on this button will allow administrators to add a new record in the table. Clicking [here](django-add-data.md) will redirect users to add data record documentation.
```
Comment (Jeremy-Prior) [inline docs/src/administrator/manual/django-table.md:7]:
```suggestion
1. **Add button**: Clicking on this button will allow administrators to add a new record in the table. Clicking [here](django-add-data.md) will redirect users to the add data record documentation.
```

**Review by Jeremy-Prior — CHANGES_REQUESTED** [review]

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 1 min 33 sec

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 1 min 21 sec

**Review by Jeremy-Prior — APPROVED** [review]
LGTM

**Comment by Jeremy-Prior** [comment]
@dimasciput please merge

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 1 min 29 sec

**Pull request merged**